### PR TITLE
Update domains.json

### PR DESCRIPTION
**Pull request by FMercedes**
#### Hi! What is your PR about?

- [ ] Add or edit a skin
- [x ] Add a custom domain for your space
- [ ] Add an alias to migrate your space

### CODE REVIEW

File: spaces/domains.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "governance.cyberestate.io": "Cyberrealestate.eth"
```
Comment (gokaiorg) [inline spaces/domains.json:2]: ENS must be in lower case. Uppercase is not allowed.
```suggestion
  "governance.cyberestate.io": "cyberrealestate.eth"
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

File: spaces/domains.json (hunk 1 above)
Comment (FMercedes) [reply spaces/domains.json]: fixed Thank you for your help I really appreciate it.

File: spaces/domains.json [hunk 2]
```diff
@@ -1,4 +1,5 @@
 {
+  "governance.cyberestate.io": "cyberrealestate.eth"
```
Comment (gokaiorg) [inline spaces/domains.json:2]: A comma is missing at the end of the line of your sub-domain.
```suggestion
  "governance.cyberestate.io": "cyberrealestate.eth",
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

File: spaces/domains.json (hunk 2 above)
Comment (FMercedes) [reply spaces/domains.json]: oh ok, I fixed it, I can't believe I missed that. Thank you for your help I appreciate it.

**Review by gokaiorg — APPROVED** [review]

**Pull request merged**